resources/aro/exploring-available-images.md

```diff
@@ -0,0 +1,35 @@
+## Check #1: Exploring the images you leverage and its availability
+
+Double check your build and deployment infrastructure for the images you are leveraging. 
+There is a __high likelihood__ that some of the images you are referencing will not exist inside your new namespace, the default `openshift` namespace etc. 
+
+> This is also a good opportunity to discover out-of-date images and upgrade them wherever possible
+
+
+### S2I builds 
+
+There is a high chance some of your s2i builder images do not exist within the namespace. You will need to rebuild these inside of your `tools` namespace and update references to the s2i builder within your `BuildConfig`(s).
```

Comment:
I think if you're using s2i, then you're already farther away from wanting to manage/support your base image lifecycle, and you want to leverage an externally managed/patched s2i base? 

your internal (to your project) image management can have tagging related to the external s2i tags as well as your code rev?

Reply:
ah that is a good point. I based this on my usage of a caddy s2i which is bcgov maintained. Most projects would be leveraging external s2is. I'll change this up